src/main/java/techreborn/blockentity/machine/multiblock/FusionControlComputer/FusionControlComputerBlockEntity.java

```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-package techreborn.blockentity.machine.multiblock;
+package techreborn.blockentity.machine.multiblock.FusionControlComputer;
```

Comment:
Why?

Reply:
So it's in a different package together with the Enum.

The alternative would be to have the enum in the class file (`FusionControlComputerBlockEntity`). 
I don't like it, but if you would prefer it, I can move it.

Reply:
You can keep it in another package, but please make it follow java conventions: `fusion` will do as the package name.

Reply:
Actually, it would probably be better to make it a nested enum called `State` (in the same file), given that it's not used in many other places.

Reply:
Fixed.